test/RibbonOptionsVault.js

```diff
@@ -265,13 +265,22 @@ function behavesLikeRibbonOptionsVault(params) {
 
       // If mintable token, then mine the token
       if (params.mintConfig) {
-        await mintToken(
-          this.assetContract,
-          params.mintConfig.contractOwnerAddress,
+        const addressToDeposit = [
           userSigner,
-          this.vault.address,
-          parseEther("200")
-        );
+          managerSigner,
+          counterpartySigner,
+          adminSigner,
+        ];
+
+        for (let i = 0; i < addressToDeposit.length; i++) {
+          await mintToken(
+            this.assetContract,
+            params.mintConfig.contractOwnerAddress,
+            addressToDeposit[i],
+            this.vault.address,
+            parseEther("200")
+          );
+        }
```

Comment:
I don't think this code block actually blocks on each index because all `await` invocations within iterations are similar to doing callbacks. This means that we will likely be passing the 0th index each time.

To fix, use Promise.all:

```js
await Promise.all(
  mintToken(...),
  mintToken(...),
  mintToken(...)
)
```